layouts/security_rules/single.html

```diff
@@ -6,11 +6,11 @@ <h1 id="pagetitle">{{ .Title }}</h1>
 	{{ partial "translate_status_banner/translate_status_banner.html" . }}
 
 	<div class="rule-specification row align-items-center mb-3">
-		{{ $meta_image := partial "security-rule-map.html" (dict "source" .Params.source "scope" .Params.scope "ctx" .) }}
+		{{ $int_logo := partial "integrations-logo.html" (dict "context" $dot "basename" (.Params.source | default .Params.scope) "variant" "avatar") }}
```

Comment:
@sarina-dd 
- I've adjusted the size on this. 
- We are relying on the app for logos now and there doesn't appear to be elbv2 version. If its needed could you ping designops to make a new one if thats needed or add an alias to elb? File Integrity Monitoring, and Runtime Security have always been the default for now. I don't believe we have established anything else for them.
- The security platform filters should be working again
- I dug into the getting started expanded bug and it turns out its because of the way i named the branch and the path matching js we have. It looks good locally and is the same code as production so it will be fine but i'll see if i can fix in the meantime anyway.

let me know if you see anything else 👍

Reply:
Thanks David! I'll ping designops about the missing logos. Everything else looks good on my end 👍